python/ray/train/_internal/backend_executor.py

```diff
@@ -111,6 +111,10 @@ def start(
             actor_cls_kwargs=train_cls_kwargs,
             placement_group=placement_group,
         )
+        print(
+            f"Starting {self._trial_info.name} on data parallel workers: "
+            f"{[w.metadata.node_ip for w in self.worker_group.workers]}."
+        )
```

Comment:
Can we use `logging` instead of `print`?

Reply:
Does `info` show up?

Reply:
hmmm, I do remember seeing info logs from actors. Changing to info then!